atomic_reactor/plugins/pre_download_remote_source.py

```diff
@@ -58,6 +64,22 @@ def run(self):
         with tarfile.open(archive) as tf:
             tf.extractall(dest_dir)
 
+        # Inject cachito provided configuration files
+        for config in self.config_files:
+            config_path = os.path.join(dest_dir, config['path'])
+            if config['type'] == CFG_TYPE_B64:
+                data = base64.b64decode(config['content'])
+                with open(config_path, 'wb') as f:
+                    f.write(data)
+            elif config['type'] == CFG_TYPE_URL:
```

Comment:
So I decided to not add this type to Cachito after all. We may have to eventually, but for now, just the base64 type is supported.

Until that support is added to Cachito, I think OSBS shouldn't expect that configuration type. We can always add it in later.

Reply:
I removed the 'url' type